src/cuda/cuda-query.cpp

```
@@ -72,21 +147,71 @@ Result PlainBufferProxyQueryPoolImpl::init()
+        return SLANG_E_INVALID_ARG;
+    }
+    QueryRangeInfo queryInfo = getQueryRangeInfo(queryIndex, count);
+    if (queryInfo.state == QueryRangeState::Reset)
+    {
+        return SLANG_FAIL;
+    }
     if (count == 0)
     {
         return SLANG_OK;
     }
 
```

Cppcheck (2.20.0)

[style] The function 'makeFromInt64' is never used.

(unusedFunction)